src/com/tw/go/plugin/util/ListUtil.java

```diff
@@ -1,30 +1,9 @@
 package com.tw.go.plugin.util;
 
-import java.util.Collection;
-import java.util.Iterator;
 import java.util.List;
 
 public class ListUtil {
     public static boolean isEmpty(List list) {
```

Comment:
I don't think we're using this method? Given this is the only one lingering in this Util, I would love to see this removed if possible 😄

Reply:
Yeah, I initially removed the whole thing; however I then saw a couple of references from https://github.com/ashwanthkumar/gocd-build-github-pull-requests/blob/master/src/main/java/in/ashwanthkumar/gocd/github/GitHubPRBuildPlugin.java#L355 and similar, so I left it around to avoid a breaking change for your other plugin. Happy to remove though If you'd rather?

Reply:
Please go ahead and remove it. This would also force me to upgrade the git-cmd version on the PR plugin.